### PR TITLE
12 add tlg section

### PR DESCRIPTION
**Pull request by lewandt**
I added the new section with the output for demographic table based on the admiral adsl data

### CODE REVIEW

File: content/output/_index.en.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+pre: "<b>1. </b>"
+chapter: true
+description: "Output and other products"
+---
+
+### Section 2
+
+# Output examples
+
+This section will contain examples generating output demographic.
```
Comment (rossfarrugia) [inline content/output/_index.en.md:13]: demographic**s**?

File: content/output/demographic/_index.Rmd [hunk 2]
```diff
@@ -0,0 +1,98 @@
+---
+title: "Create Demographic table"
+author: "Tadeusz Lewandowski"
+date: "16/08/2022"
+output: html_document
+description: "An example of how to generate an Demogrphic output based on ADaM dataset"
```
Comment (rossfarrugia) [inline content/output/demographic/_index.Rmd:6]: Demogr**a**phic

File: content/output/demographic/_index.Rmd [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+---
+
+## Introduction
+
+This guide will show you how two pharmaverse packages, along with some from tidyverse, can be used to create a Demographic table as an end-to-end, using the Admiral ADSL data as input.
```
Comment (rossfarrugia) [inline content/output/demographic/_index.Rmd:11]: ...the `{admiral}` `ADSL` data...

File: content/output/demographic/_index.Rmd [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+The two packages used with a brief description of their purpose are as follows:
+
+* [`{rtables}`](https://roche.github.io/rtables/index.html): designed to create and display complex tables with R.
```
Comment (rossfarrugia) [inline content/output/demographic/_index.Rmd:15]: Should we mention here some sentence like:
Alternative tabulation packages exist that could be considered here instead such as `{Tplyr}` (see https://pharmaverse.org/e2eclinical/tlg/)

File: content/output/demographic/_index.Rmd [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+```{r}
+adsl <- adsl %>%
+  mutate(
+    SEX = factor(case_when(
+      SEX == "M" ~ "Male",
+      SEX == "F" ~ "Female",
+      SEX == "U" ~ "Unknown",
+      SEX == "UNDIFFERENTIATED" ~ "Undifferentiated"
+    )),
+    AGEGRP = factor(case_when(
```
Comment (rossfarrugia) [inline content/output/demographic/_index.Rmd:59]: we already have AGEGR1 derived in the ADSL example, so could we use this directly? re-formatting variables for display is fine but i wouldn't want people to think we advocate derivations in TLG programs, as these would be done in ADaM

File: content/output/demographic/_index.Rmd [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+
+var_labels <- c(
+  "Age (yr)",
+  "Age groups",
+  "Sex",
+  "Race"
+)
+```
+
+Now we create the demographic table using the [`{rtables}`](https://roche.github.io/rtables/index.html) and [`{tern}`](https://insightsengineering.github.io/tern/main/index.html) packages.
```
Comment (rossfarrugia) [inline content/output/demographic/_index.Rmd:83]: Maybe add: ...using `ARM` as the grouping variable

File: content/output/demographic/_index.Rmd [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+## Introduction
+
+This guide will show you how two pharmaverse packages, along with some from tidyverse, can be used to create a Demographic table as an end-to-end, using the Admiral ADSL data as input.
+
+The two packages used with a brief description of their purpose are as follows:
+
+* [`{rtables}`](https://roche.github.io/rtables/index.html): designed to create and display complex tables with R.
+* [`{tern}`](https://insightsengineering.github.io/tern/main/index.html): contains analysis functions to create tables and graphs used for clinical trial reporting.
+
+It is important to understand [`{rtables}`](https://roche.github.io/rtables/index.html) grammar by reading through the above linked package site, as these are fundamental to being able to use [`{tern}`](https://insightsengineering.github.io/tern/main/index.html).
```
Comment (rossfarrugia) [inline content/output/demographic/_index.Rmd:18]: Would we be able to add a section on how to bring this display into an IDD module via teal or would you say best to do that as a separate example?

**Review by rossfarrugia — CHANGES_REQUESTED** [review]
@lewandt thanks for building this! i added some review comments for consideration

File: content/output/demographic/_index.Rmd (hunk 4 above)
Comment (lewandt) [reply content/output/demographic/_index.Rmd:15]: I will have a separate example for Tplyr with the ad-hoc module. For simplicity of the code, I aimed to present the same static output also in the Shiny using the teal.module.clinical.

File: content/output/demographic/_index.Rmd (hunk 5 above)
Comment (lewandt) [reply content/output/demographic/_index.Rmd:59]: Ok good point, I can amend it, but will leave some code to present that the pre-processing could be done at the output level program. As the same example I have for the Shin, so I will synchronize booth codes.

File: content/output/_index.en.md (hunk 1 above)
Comment (lewandt) [reply content/output/_index.en.md]: Done

File: content/output/demographic/_index.Rmd (hunk 2 above)
Comment (lewandt) [reply content/output/demographic/_index.Rmd]: Done

File: content/output/demographic/_index.Rmd (hunk 7 above)
Comment (lewandt) [reply content/output/demographic/_index.Rmd:18]: I will have a separate example for APPs

File: content/output/demographic/_index.Rmd (hunk 3 above)
Comment (lewandt) [reply content/output/demographic/_index.Rmd]: Done

**Comment by lewandt** [comment]
Changes amended

**Review by lewandt — COMMENTED** [review]
I amended your comments thank you

**Review by rossfarrugia — APPROVED** [review]
all sounds good - thanks @lewandt

**Pull request merged**